src/frontend/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Capacity Management" />
-    <link rel="stylesheet" href="/libraries/nhsuk.css" />
+    <link rel="stylesheet" href="/main.scss" />
```

Comment:
You can replace this with importing the scss file directly inside of the base `index.tsx` - also helps keep track for live reloading of styles in dev mode.